indico/http/client.py

```diff
@@ -81,12 +81,20 @@ def _handle_files(self, req_kwargs):
         new_kwargs = deepcopy(req_kwargs)
         files = []
         file_arg = {}
+        dup_counts = {}
         if "files" in new_kwargs:
             for filepath in new_kwargs["files"]:
                 path = Path(filepath)
                 fd = path.open("rb")
                 files.append(fd)
-                file_arg[path.stem] = fd
+                # follow the convention of adding (n) after a duplicate filename
+                if path.stem in dup_counts:
+                    file_arg[path.stem+f"({dup_counts[path.stem]})"] = fd
```

Comment:
Can we avoid changing the filenames at all. That will break string compares on the customer side.

Reply:
this is internal-only - please see above - the storage objects that come back have the same name passed in - no change